### PR TITLE
Fixed missing sections and descriptions for cheatsheet 2

### PR DESCRIPTION
**Pull request by gautamkrishnar**
fixed:
* missing sections
* descriptions

for passing prove test:
prove t/CheatSheets/CheatSheetsJSON.t

fixes:#2015

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by gautamkrishnar** [comment]
Don't know why this JSON files show warning even though everything looks fine:
```bash
# Testing php...........OK                                                                                                                                                                    
#        -> WARN: Expected 'php array functions' but not found                                                                                                                                
# Testing php...........OK                                                                                                                                                                    
#        -> WARN: Expected 'php string functions' but not found                                                                                                                               
# Testing php...........OK                                                                                                                                                                    
#        -> WARN: Expected 'regular expressions syntax' but not found      

# Testing sms...........OK                                                                                                                                                                    
#        -> WARN: Expected '0-9' but not found 
```

File: share/goodie/cheat_sheets/json/vlc.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
     "id": "vlc_cheat_sheet",
     "name": "VLC Media Player Keyboard Shortcuts",
+    "description":"It is a free and open source cross-platform multimedia player and framework",
```
Comment (MrChrisW) [inline share/goodie/cheat_sheets/json/vlc.json]: `VLC media player is a portable, free and open-source, cross-platform media player and streaming media server`

File: share/goodie/cheat_sheets/json/phpstorm-osx.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "id": "phpstorm_osx_cheat_sheet",
     "name": "JetBrains PHPStorm IDE (OS X)",
-    "descriptions": "Keyboard shortcuts for OS X version PHPStorm IDE, a PHP IDE developed by JetBrains",
+    "description": "Keyboard shortcuts for OS X version PHPStorm IDE, a PHP IDE developed by JetBrains",
```
Comment (MrChrisW) [inline share/goodie/cheat_sheets/json/phpstorm-osx.json]: Keyboard shortcuts for OS X version **of** PHPStorm IDE, a PHP IDE developed by JetBrains

File: share/goodie/cheat_sheets/json/vlc.json (hunk 1 above)
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/vlc.json]: :+1:

File: share/goodie/cheat_sheets/json/html.json [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
     "id": "html_cheat_sheet",
     "name": "HTML Elements",
+    "description":"Hyper text markup language is a language for describing websites",
```
Comment (MrChrisW) [inline share/goodie/cheat_sheets/json/html.json]: HyperText Markup Language, is the standard markup language used to create web pages.

File: share/goodie/cheat_sheets/json/phpstorm-osx.json (hunk 2 above)
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/phpstorm-osx.json]: :+1:

File: share/goodie/cheat_sheets/json/html.json (hunk 3 above)
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/html.json]: :+1:

File: share/goodie/cheat_sheets/json/elementary.json [hunk 4]
```diff
@@ -1,6 +1,7 @@
 {
     "id": "elementary_cheat_sheet",
     "name": "elementary OS",
+    "description":"A fast and open replacement for Windows and OS X",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/elementary.json]: Space between `:` and `"A` please (I'm picky ;))

File: share/goodie/cheat_sheets/json/html.json [hunk 5]
```diff
@@ -1,6 +1,7 @@
 {
     "id": "html_cheat_sheet",
     "name": "HTML Elements",
+    "description":"HyperText Markup Language, is the standard markup language used to create web pages",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/html.json]: And here.

File: share/goodie/cheat_sheets/json/processing-lang.json [hunk 6]
```diff
@@ -1,6 +1,7 @@
 {
     "id": "processing_lang_cheat_sheet",
     "name": "Processing",
+    "description":"Processing is a flexible language for learning how to code within the context of the visual arts",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/processing-lang.json]: On this one too - just check over the descriptions, there's a few that need doing.

File: share/goodie/cheat_sheets/json/elementary.json (hunk 4 above)
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/elementary.json]: :smile:

**Comment by gautamkrishnar** [comment]
@moollaza pls check this out

**Comment by moollaza** [comment]
@gautamkrishnar Thanks! This LGTM :+1:

**Pull request merged**